src/Calendar.Plugin/Shared/Controls/Calendar.xaml.cs

```diff
@@ -195,6 +195,15 @@ public Color ArrowsColor
             set => SetValue(ArrowsColorProperty, value);
         }
 
+        public static readonly BindableProperty ArrowVisibleProperty =
+            BindableProperty.Create(nameof(ArrowVisible), typeof(bool), typeof(Calendar), true);
+
+        public bool ArrowVisible
```

Comment:
`ArrowVisible` sounds too generic, and especially since we have a few more arrows in the Header section
`FooterArrowVisible` might be better

also if it is not too much trouble for you we could also add `FooterSectionVisible` to be used to completely hide footer section - https://github.com/lilcodelab/Xamarin.Plugin.Calendar/issues/17

Reply:
@PenguinPero I agree it was too generic . I've made the changes, please review.